### ISSUE TITLE
KeyError: 'notebookBackend'

### ISSUE THREAD

**Issue by NataliaFavila** [brainglobe/bg-heatmaps#7]
When running the example from heatmap_2D.py:
import bgheatmaps as bgh

"""
    This example shows how to use visualize a heatmap in 2D
"""

values = dict(  # scalar values for each region
    TH=1,
    RSP=0.2,
    AI=0.4,
    SS=-3,
    MO=2.6,
    PVZ=-4,
    LZ=-3,
    VIS=2,
    AUD=0.3,
    RHP=-0.2,
    STR=0.5,
    CB=0.5,
    FRP=-1.7,
    HIP=3,
    PA=-4,
)

f = bgh.heatmap(
    values,
    position=5000,  # when using a named orientation you can pass a single value!
    orientation="frontal",  # 'frontal' or 'sagittal', or 'horizontal' or a tuple (x,y,z)
    title="horizontal view",
    vmin=-5,
    vmax=3,
    format="2D",
).show()

I get the following Error:

File ~/opt/anaconda3/envs/py3.8/lib/python3.8/site-packages/vedo/utils.py:238, in dotdict.__getattr__(self, k)
    237 def __getattr__(self, k):
--> 238     return self[k]

KeyError: 'notebookBackend'

**Comment by marcomusy** [comment]
The latest `vedo` version is no more permissive when trying to access a non-existing key in a `dotdict` (dotted dictionary).
You may need to temporarily downgrade to 
`pip install vedo==2022.0.1`
unitl it's fixed in `brainrender` or `bg-heatmaps`.

**Comment by FedeClaudi** [comment]
As Marco suggests, the solution is downgrade vedo with `pip install vedo==2021.0.5`.

Brainrender will only support that version of vedo for the time being.

-------

@marcomusy 

I was going to make brainrender compatible with the newest vedo, but I noticed a number of changes. The camera orientation and position was different, the direction corresponding to the normal of planes changed, the position and orientation of the axes changed, how notebooks are handled changed...
What's causing all these changes?

In the end I've decided that since I'm not planning to develop brainrender any further I will simply pin it to the last working version of vedo so that there should be these kind of issues in the future. Beyond brainrender `vedo` is currently not used much by the neuroscientific community (unfortunately) so this shouldn't create problems for users.
Thank you for your help!

**Comment by FedeClaudi** [comment]
And please [NataliaFavila](https://github.com/NataliaFavila) let meknow if this helps and how you're getting on with using `bg-heatmaps`!

**Comment by marcomusy** [comment]
@FedeClaudi 
sorry for having caused you troubles :)

> I was going to make brainrender compatible with the newest vedo, but I noticed a number of changes. The camera orientation and position was different, 

> the direction corresponding to the normal of planes changed, the position and orientation of the axes changed, 

This is quite strange, I cannot think right now of a good reason why that should have changed..

> how notebooks are handled changed...

Yes! indeed that has changed ... but the transition should be reasonably smooth.... if not at all backward compatible..

I will have a go looking at the code, as soon as i can, to see if I understand any of the differences you observe.
Thanks for your feedback and patience!

**Comment by FedeClaudi** [comment]
That's okay! Vedo is amazing and its impressive how much it keeps growing!

If I needed to continue adding new features/developing brainrender I would happily keep up, but since it's fairly complete as a project I'm happy to just pin it to a working version.

For reference, I had to change the sign of the `viewup` vectors and the position of the [cameras](https://github.com/brainglobe/brainrender/blob/master/brainrender/cameras.py), otherwise everyhing was upside/down. 
I didn't look closely into this so not sure if the camera was off, the origin or if somehow the meshes were shown in some weird orientation. Might be useful for you to figure out but its not a problem at all for brainrender since I'll be using the older vedo.

Once again, thank you so much for all your help. Brainrender wouldn't exist without you!
Wish you all the best and great success with vedo
